automation/README.md

```diff
@@ -46,21 +69,21 @@ order to run the tests. Setting the environment variable `debug_exit_shell`
 will make the script start a shell instead of exiting the script after an error
 or running the scripts:
 
-`debug_exit_shell=1 ./automation/run-integration-tests.sh`
+`debug_exit_shell=1 ./automation/run-tests.sh`
 
 After closing the shell, the container will be removed. Alternatively it is
 possible to provide the `--debug-shell` command-line option.
 
 To specify a different container image for the tests, specify it with the
 `DOCKER_IMAGE` variable:
 
-`DOCKER_IMAGE=local/centos7-nmstate-dev debug_exit_shell=1 ./automation/run-integration-tests.sh`
+`DOCKER_IMAGE=local/fedora-nmstate-dev debug_exit_shell=1 ./automation/run-tests.sh`
```

Comment:
I the fedora image is the default now, perhaps you should mention here centos7.

Reply:
Done. Thanks.